### DISCUSSION TITLE
icon_picker multiple instance problem

### DISCUSSION THREAD

**Discussion by siberfx** [Laravel-Backpack/community-forum#738] [Q&A (Help)]
when it comes to build in same page multiple icon_picker instances it doesnt work, also when its inside tabbed view same matter happens. is there anyone who had the issue ? or the solution ?

**Comment by tabacitu** [comment]
Hi @siberfx ,

What do you mean by "not working"? Do you get console errors? Does it look broken? Does it not show at all? 

1. I see it's working in tabs in our online demo - https://demo.backpackforlaravel.com/admin/monster/create#miscellaneous

2. And I've just tried adding two `icon_picker` fields in one tab and that worked for me too:

![CleanShot 2023-11-09 at 12 36 33](https://github.com/Laravel-Backpack/community-forum/assets/1032474/e33266fe-2eed-4d65-b466-1f65528673fb)

So I cannot reproduce or confirm any of the two bugs you reported 👀

More info please on what you are doing and what you are seeing?

**Comment by siberfx** [comment]
no console errors, just they turn into usual inputs I dont know the reason

**Reply by tabacitu** [reply]
Hmmm that's odd. Are you using a custom layout blade file? Sounds like the JS isn't loading onpage.

**Reply by siberfx** [reply]
nope, not at all, laravel 10, backpack 6 all js loads well. by the way; I think its all about these are under repeatable subfields, possible its the case

**Reply by tabacitu** [reply]
Oooohh. Any chance you can provide a video recording of your screen when having the problem? Should help us get more reasons it could be happening.
